config.toml

```
@@ -149,6 +149,13 @@ weight = 11
 [languages.fr.params]
 description = "Le projet de Glossaire CNCF Cloud Native a pour objectif de référencer les termes courants utilisés dans les différentes applications Cloud Natives."
 
+[languages.ru]
+title = "Глоссарий Cloud Native"
+languageName ="Русский (Russian)"
+weight = 11
```

Based on all other items in this config, I guess the value should be incremented.
```suggestion
weight = 12
```